port/common/omrmem32helpers.c

```diff
@@ -438,15 +430,28 @@ allocate_memory32(struct OMRPortLibrary *portLibrary, uintptr_t byteAmount, cons
 #endif
 		omrthread_monitor_enter(PPG_mem_mem32_subAllocHeapMem32.monitor);
 
-		/* Check if byteAmount is larger than HEAP_SIZE_BYTES.
+		/* Check if byteAmount is larger than PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize.
 		 * The majority of size requests will typically be much smaller.
 		 */
 		returnPtr = iterateHeapsAndSubAllocate(portLibrary, byteAmount);
 		if (NULL == returnPtr) {
-			if (byteAmount >= HEAP_SIZE_BYTES) {
-				returnPtr = allocateLargeRegion(portLibrary, byteAmount, callSite, 0);
+			if (byteAmount >= PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize) {
+				returnPtr = allocateLargeRegion(portLibrary,
```

Comment:
First parameter should be on a new line: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line
```suggestion
				returnPtr = allocateLargeRegion(
								portLibrary,
```